torchvision/prototype/transforms/functional/_meta.py

```diff
@@ -24,7 +35,17 @@ def get_dimensions(image: Union[features.ImageTypeJIT, features.VideoTypeJIT]) -
         return get_dimensions_image_pil(image)
 
 
-get_num_channels_image_tensor = _FT.get_image_num_channels
+def get_num_channels_image_tensor(image: torch.Tensor) -> int:
+    chw = image.shape[-3:]
+    ndims = len(chw)
+    if ndims == 3:
```

Comment:
Can we define numbers as constants ? What if ndims = 4 ?

Reply:
JIT doesn't like constants :( See error at: https://github.com/pytorch/vision/pull/6828#discussion_r1003613897